Solutions/PaloAlto-PAN-OS/Hunting Queries/Palo Alto - potential beaconing detected.yaml

```diff
@@ -58,5 +61,5 @@ entityMappings:
     fieldMappings:
       - identifier: Address
         columnName: IPCustomEntity
-version: 1.0.2
+version: 1.0.3
 kind: Scheduled
```

Comment:
I think ,we cant use this : kind: Scheduled in hunting query template

Reply:
Resolved